internal/attestation/crafter/testdata/contracts/with_rego.yaml

```diff
@@ -0,0 +1,4 @@
+schemaVersion: "v1"
+policies:
+  attestation:
```

Comment:
in the future we should also have some examples with `material` attachments, all of them in this directory seem to be attestation.